pallets/parachain-staking/src/mock.rs

```diff
@@ -353,9 +357,10 @@ impl ExtBuilder {
 }
 
 /// Compare whether the difference of both sides is at most `precision * left`.
+#[allow(clippy::arithmetic_side_effects)]
```

Comment:
I can't seem to find this clippy lint anywhere. What does it enforce? Can you post a link?